src/components/common/Certify.vue

```diff
@@ -64,6 +74,9 @@ export default class Certify extends Vue {
 
   @Getter(useRootStore) getCurrentDate!: string
 
+  /** Is Staff prop. */
+  @Prop({ default: false }) readonly isStaff!: boolean
```

Comment:
Since this component already accesses the store, why not just add the getter here and simplify?

Reply:
Got it, will update

Reply:
Updated, is it ready to merge now?